pandas/computation/tests/test_eval.py

```diff
@@ -50,11 +50,7 @@ def _eval_single_bin(lhs, cmp1, rhs, engine):
         try:
             return c(lhs, rhs)
         except ValueError as e:
-            try:
-                msg = e.message
-            except AttributeError:
-                msg = e
-            msg = u(msg)
+            msg = u(str(e))
```

Comment:
https://github.com/pydata/pandas/issues/12178

though I closed it. I thought was already fixed (put in a separate PR if this change is needed).